src/main/java/frc/robot/scoreStates/ShootState.java

```diff
@@ -44,20 +60,49 @@ public Command driveToClosestOptimalPoint() {
                     return AutoBuilder.pathfindToPose(
                             new Pose2d(
                                     optimalTranslation,
-                                    Utils.calculateOptimalRotation(
+                                    Utils.calcRotationToTranslation(
                                             optimalTranslation, speakerPose)),
                             Constants.AUTO_CONSTRAINTS);
                 },
                 Set.of(SwerveDrive.getInstance()));
     }
 
     @Override
-    public Command stateInitialize() {
-        return null;
+    public Command initializeCommand() {
+        return Commands.none();
+    }
+
+    @Override
+    public Command initializeSubsystem() {
+        return Commands.parallel(
+                shooter.setVelocity(
+                        () ->
+                                Units.RotationsPerSecond.of(
+                                                ShooterConstants.interpolationMap.get(
+                                                                Utils.getDistanceFromPoint(
+                                                                        speakerPose,
+                                                                        PoseEstimation.getInstance()
+                                                                                .getEstimatedPose()))
+                                                        .value)
+                                        .mutableCopy()),
+                hood.setAngle(
+                        () ->
+                                Units.Degrees.of(
+                                                Math.atan(
+                                                        (Constants.SPEAKER_TARGET_POSE.getY()
+                                                                        - ShooterConstants
+                                                                                .SHOOTER_HEIGHT)
+                                                                / Utils.getDistanceFromPoint(
+                                                                        speakerPose,
+                                                                        PoseEstimation.getInstance()
+                                                                                .getEstimatedPose())))
+                                        .mutableCopy()),
+                CommandGroups.getINSTANCE().elevatorGripperMinPosition());
     }
 
     @Override
     public Command score() {
-        return null;
+        return Commands.sequence(
```

Comment:
Just use `andThen()` for two commands.

Reply:
Was changed in 60aa413

Reply:
Didn't we say overrides will be controlled with an additional button? I fear having it as a toggle might lead to mistakes where the override wasn't clear, and it was applied to a command it shouldn't have.

Reply:
That's fine, what do you think what I said about the toggle?

Reply:
I agree, We just said we will implement it later

Reply:
No problem. Opened #46 as a follow-up.